src/include/parser/create_function_statement.h

```diff
@@ -0,0 +1,136 @@
+//===----------------------------------------------------------------------===//
+//
+//                         Peloton
+//
+//create_function_statement.h
+//
+// Identification: src/include/parser/create_function_statement.h
+//
+// Copyright (c) 2015-16, Carnegie Mellon University Database Group
+//
+//===----------------------------------------------------------------------===//
+
+#pragma once
+
+#include "expression/abstract_expression.h"
+#include "type/types.h"
+#include "common/sql_node_visitor.h"
+#include "parser/sql_statement.h"
+
+namespace peloton {
+namespace parser {
+
+struct Parameter {
```

Comment:
We kept it consistent with the other ParseNode classes present in Peloton currently, which use struct.